### PR TITLE
feat(save-load-validation): Invalid Load Assessment Dialog

### PR DESCRIPTION
**Pull request by JGibson2019**
#### Details
Added the dialog that shows when users try to load an invalid file

##### Motivation
We need to show users an error dialog when the extension can't upload the file so they can proceed properly. 

##### Context
a basic dialog that says "Oops! Accessibility Insights can't open this file." and has a subtitle that reads "Please select a valid *.a11yAssessmentfile" with a blue "Ok" button

We added a basic dialog that says "Oops! Accessibility Insights can't open this file." and has a subtitle that reads "Please select a valid *.a11yAssessmentfile" with a blue "Ok" button that dismisses the dialog. Also, if users click outside of the dialog, it will dismiss.  Image of the component described above is below

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [X] Ran `yarn fastpass`
- [X] Added/updated relevant unit test(s) (and ran `yarn test`)
- [X] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [X] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [X] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/components/invalid-load-assessment-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+
+        return (
+            <Dialog
+                hidden={false}
+                onDismiss={props.onClose}
+                dialogContentProps={{
+                    type: DialogType.normal,
+                    title: 'Oops! Accessibility Insights can’t open this file.',
+                    subText: 'Please select a valid *.a11yAssessment file',
```
Comment (brocktaylor7) [inline src/DetailsView/components/invalid-load-assessment-dialog.tsx:27]: So this text matches the text in the requirements perfectly, but the file extension is actually ".a11ywebassessment" so we should probably match that!

File: src/DetailsView/components/load-assessment-helper.ts [hunk 2]
```diff
@@ -33,7 +34,7 @@ export class LoadAssessmentHelper {
                 parsedAssessmentData = this.assessmentDataParser.parseAssessmentData(content);
             } catch {
                 console.log('Invalid JSON');
```
Comment (brocktaylor7) [inline src/DetailsView/components/load-assessment-helper.ts:36]: We should probably remove the console logs on this and the one below

File: src/DetailsView/components/load-assessment-helper.ts [hunk 3]
```diff
@@ -43,7 +44,7 @@ export class LoadAssessmentHelper {
             if (!validationData.dataIsValid) {
                 console.log('Invalid JSON schema');
```
Comment (brocktaylor7) [inline src/DetailsView/components/load-assessment-helper.ts:45]: these 2 console logs should also probably go.

**Review by brocktaylor7 — APPROVED** [review]
Left a few comments but they're all really low priority, so I've approved this and you can just push up those small changes and then squash and merge! :)

File: src/DetailsView/components/invalid-load-assessment-dialog.tsx (hunk 1 above)
Comment (JGibson2019) [reply src/DetailsView/components/invalid-load-assessment-dialog.tsx]: changed!

File: src/DetailsView/components/load-assessment-helper.ts (hunk 2 above)
Comment (JGibson2019) [reply src/DetailsView/components/load-assessment-helper.ts]: removed

**Pull request merged**